sw/device/tests/pmod/BUILD

```diff
@@ -311,18 +313,20 @@ opentitan_test(
 opentitan_test(
     name = "i2c_host_fram_test",
     srcs = ["i2c_host_fram_test.c"],
-    exec_env = {
-        "//hw/top_earlgrey:fpga_cw310_sival": None,
-        "//hw/top_earlgrey:silicon_owner_sival_rom_ext": "silicon_owner",
-        "//hw/top_earlgrey:fpga_cw310_test_rom": None,
-    },
+    exec_env = dicts.add(
+        EARLGREY_TEST_ENVS,
+        EARLGREY_SILICON_OWNER_ROM_EXT_ENVS,
+        {
```

Comment:
Would drop it.

Reply:
Will do in a follow up since there are many tests that may have unnecessary `fpga_cw310_sival` environments.

It's not doing any harm either since it's not run in CI if `fpga_cw310_sival_rom_ext` is present.